### DISCUSSION TITLE
Multiple vacuums settings

### DISCUSSION THREAD

**Discussion by Cikkajes** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#121] [Q&A] [answered]
Hello, I have 2 roborocks and I have installed both the map card and the map extractor addons. I am now setting up the card on lovelace but I am not sure it automatically knows wich map to upload based on the entity name. What are the recommended settings for 2+ roborocks? Thanks.

EDIT: solved by updating the map_camera field. The name was the generic name given in the template.

**Comment by PiotrMachowski** [comment, marked as answer]
Applicable to v1.x.x
Answer in "EDIT" section
